sdk/src/main/java/com/hcaptcha/sdk/HCaptchaDialogFragment.java

```diff
@@ -166,9 +164,24 @@ public void onAnimationEnd(Animator animation) {
         }
     }
 
+    @Override
+    public void onLoaded() {
+        assert webViewHelper != null;
+
+        // If the size is `invisible` - wait until challenge is shown and handle it in the `onOpen` callback
```

Comment:
Comment is redundant.

Reply:
Done in https://github.com/hCaptcha/hcaptcha-android-sdk/pull/78/commits/a3f5189bf27feeb1061ddc1b1d6763a91b9a9a93